lib/index.js

```diff
@@ -25,9 +25,9 @@ module.exports.attachOnDuplicateUpdate = function attachOnDuplicateUpdate() {
 
       return result;
     }, { placeholders: [], bindings: [] });
-
+    const escapeKnexString = input => input.split('?').join('\\?');
     return this.client.raw(
-      `${this.toString()} on duplicate key update ${placeholders.join(', ')}`,
+      `${escapeKnexString(this.toString())} on duplicate key update ${placeholders.join(', ')}`,
```

Comment:
instead of using `split`&`join` I think that it is better to use `.replace(/\?/g, '\\?')`

WDYT?

Reply:
I did up a quick jsperf and it looks like split / join is faster by a large margin on Firefox but slower by about 25% on Safari and Chrome.  I personally think that split / join is more readable but this is your project so let me know which one you would prefer.  

https://jsperf.com/splitvsregexforreplacement

Reply:
I went ahead and just changed over to regex since it's more consistently quicker.